edge-modules/TwinTester/Settings.cs

```diff
@@ -20,47 +20,70 @@ public class Settings
                     .AddEnvironmentVariables()
                     .Build();
 
-                string moduleId = configuration.GetValue<string>("TargetModuleId");
-                moduleId = string.IsNullOrWhiteSpace(moduleId) ? configuration.GetValue<string>("IOTEDGE_MODULEID", string.Empty) : moduleId;
+                string moduleId = configuration.GetValue<string>("IOTEDGE_MODULEID", string.Empty);
+                string targetModuleId = configuration.GetValue<string>("TargetModuleId", moduleId);
+
+                string analyzerUrl = configuration.GetValue<string>("AnalyzerUrl");
+                string testResultCoordinatorUrl = configuration.GetValue<string>("testResultCoordinatorUrl");
```

Comment:
Is the TwinTester going to be part of the Long Haul and Stress tests? I'm thinking we probably can have a default endpoint here.

Reply:
yes, added the default and changed code to first check if trcUrl is set, otherwise use analyzer

---
In reply to: [361727022](https://github.com/Azure/iotedge/pull/2177#discussion_r361727022) [](ancestors = 361727022)